### PR TITLE
Create Project Marvel

### PR DESCRIPTION
**Pull request by HyperionsThs**

### CODE REVIEW

File: MarvelApplication/.gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+*.iml
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
```
Comment (daolq3012) [inline MarvelApplication/.gitignore]: Please change 
```/.idea/workspace.xml
 /.idea/libraries``
=>>
```
/.idea
```

File: MarvelApplication/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        <activity android:name=".MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN"/>
+
+                <category android:name="android.intent.category.LAUNCHER"/>
+            </intent-filter>
+        </activity>
+    </application>
+
+</manifest>
```
Comment (daolq3012) [inline MarvelApplication/app/src/main/AndroidManifest.xml]: enter when end of file

Comment (HyperionsThs) [inline MarvelApplication/app/src/main/AndroidManifest.xml]: phải enter như trên mới đúng hay không được enter cuối dòng vậy a

File: MarvelApplication/.gitignore [hunk 3]
```diff
@@ -0,0 +1,9 @@
+*.iml
+.gradle
+/local.properties
+/.idea/workspace.xml
```
Comment (daolq3012) [inline MarvelApplication/.gitignore]: Please remove this line

File: MarvelApplication/.idea/compiler.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (daolq3012) [inline MarvelApplication/.idea/compiler.xml]: Please remove this file from commit

File: MarvelApplication/.idea/copyright/profiles_settings.xml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<component name="CopyrightManager">
```
Comment (daolq3012) [inline MarvelApplication/.idea/copyright/profiles_settings.xml]: Please remove this file from commit

File: MarvelApplication/.idea/gradle.xml [hunk 6]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (daolq3012) [inline MarvelApplication/.idea/gradle.xml]: Please remove this file from commit

File: MarvelApplication/.idea/misc.xml [hunk 7]
```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (daolq3012) [inline MarvelApplication/.idea/misc.xml]: Please remove this file from commit

File: MarvelApplication/.idea/modules.xml [hunk 8]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (daolq3012) [inline MarvelApplication/.idea/modules.xml]: Please remove this file from commit

File: MarvelApplication/.idea/runConfigurations.xml [hunk 9]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (daolq3012) [inline MarvelApplication/.idea/runConfigurations.xml]: Please remove this file from commit

**Pull request closed without merging**